app/src/main/java/com/example/easysurvey/ui/theme/Type.kt

```diff
@@ -13,16 +13,5 @@ val Typography = Typography(
                 fontWeight = FontWeight.Normal,
                 fontSize = 16.sp
         )
-        /* Other default text styles to override
-    button = TextStyle(
-        fontFamily = FontFamily.Default,
-        fontWeight = FontWeight.W500,
-        fontSize = 14.sp
-    ),
-    caption = TextStyle(
```

Comment:
Comments were not being used and therefore increasing technical debt, cited from https://wakeupandcode.com/todo-comments-and-technical-debt/ 
https://kotlinlang.org/docs/coding-conventions.html#source-code-organization

Reply:
Good.